docs/integrators/snapshotless-observing-squad.md

```diff
@@ -0,0 +1,68 @@
+---
+id: snapshotless-observing-squad
+title: Snapshotless Observing Squad
+---
+
+[comment]: # (mx-abstract)
+
+This page describes the Snapshotless Observing Squad, a type of Observing Squad optimized for real-time requests such as accounts data fetching and vm-query operations.
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+Whenever a node is executing the trie snapshotting process, the accounts data fetching & vm-query operations are greatly affected.
+This is caused by the fact that the snapshotting operation consumes a large number of CPU cycles along with disk iops.
+The nodes started with the flag `--operation-mode snapshotless-observer` will not create trie snapshots on every epoch and will 
+also prune the trie storage in order to save space. 
```

Comment:
There's an unnecessary newline above (markdown renderer will ignore it, anyway).

Reply:
just to break the code in the IDE

Reply:
removed cessary new-lines